test/Datadog.Trace.TestHelpers/EnvironmentHelper.cs

```diff
@@ -166,9 +168,11 @@ public static void ClearProfilerEnvironmentVariables()
                 environmentVariables["COR_ENABLE_PROFILING"] = "1";
                 environmentVariables["COR_PROFILER"] = EnvironmentHelper.ProfilerClsId;
                 environmentVariables["COR_PROFILER_PATH"] = GetProfilerPath();
+
+                processName = Path.GetFileName(processPath);
```

Comment:
Why do we only set processName to the filename for .NET Framework and not for CoreCLR?

Reply:
They're different because in .NET Core the process name is always `dotnet.exe` on Windows or `dotnet` on Linux (at least the way we build the sample projects). I'm not sure what's going on here, though.

Reply:
At some point with my test refactors, when I was focused on getting the core ones going, I neglected the framework ones. The full file path comes in for the framework tests and the profiler never successfully picks up the exe we are looking to test.

Reply:
The process name can be different things depending on the app been profiled:
- app.exe (netfx, or netcore depending on how the app is published)
- w3wp.exe (iis) or iisexpress.exe
- dotnet.exe (Windows) or dotnet (Linux) for netcore command line or
out-of-process iis[express].

Note that for netcore on iis[express], the process name can be the hosting
process (iisexpress.exe) if running in-process or dotnet.exe if running out
of process.

But again, we should be able to drop this entirely. The only reason I added
it was to start VS with the env vars set (so iisexpress can inherit them),
but avoid profiling VS itself.